benchmark-ganache.ts

```diff
@@ -0,0 +1,165 @@
+import * as assert from "assert";
+
+const chalk: any = require("chalk");
+import * as ethers from "ethers";
+import Ganache from "ganache";
+
+const url = "mainnet";
+const blockGasLimit = "0x2625A00"; // 40,000,000
```

Comment:
Shouldn't make a difference, but just curious why 40M and not the current block gas limit of 30M? (and same question below where we use a 39M tx gas limit)

Reply:
No real reason... I had it set to 20M and that failed so I doubled it :)

Updated this to use 30M, and also I changed the tx gasLimit to match the blockLimit, since my caution to use a lower value appears now to be unnecessary.